typescript/sdk/src/gas/HyperlaneIgpDeployer.ts

```diff
@@ -73,7 +74,9 @@ export class HyperlaneIgpDeployer extends HyperlaneDeployer<
       }
     }
 
+    // Okay so the deployer is not the owner of the IGP, for some reason...
     if (gasOracleConfigsToSet.length > 0) {
+      this.logger({ gasOracleConfigsToSet });
```

Comment:
clean this?

Reply:
Ugh, that's what I get for grepping for console.log